tests/src/actionContainers/Swift3ActionContainerTests.scala

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
I added CLI test for wsk action create --kind, also added a KituraNet/Dispatch test.  Swift3 should also repeat swift tests.